Mathlib/Geometry/Euclidean/Angle/Unoriented/TriangleInequality.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright (c) 2025 Ilmārs Cīrulis. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ilmārs Cīrulis, Alex Meiburg
+-/
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.Affine
+import Mathlib.Geometry.Euclidean.Angle.Unoriented.UnitVectorAngles
+import Mathlib.Analysis.NormedSpace.Normalized
+
+/-!
+# The Triangle Inequality for Angles
+
+This file contains proof that angles obey the triangle inequality.
+-/
+
+open InnerProductGeometry
+
+open scoped RealInnerProductSpace
+
+variable {V : Type*}
+variable [NormedAddCommGroup V]
+variable [InnerProductSpace ℝ V]
+
+@[simp]
+private lemma angle_normalized_left (x y : V) :
+    angle (normalized x) y = angle x y := by
+  by_cases hx : x = 0
+  · simp [hx]
+  replace hx : 0 < ‖x‖⁻¹ := by simp [hx]
+  simp only [normalized, angle_smul_left_of_pos, hx, angle_smul_right_of_pos]
+
+@[simp]
+private lemma angle_normalized_right (x y : V) :
+    angle x (normalized y) = angle x y := by
+  rw [angle_comm, angle_normalized_left, angle_comm]
```

Comment:
I think these belong in `Mathlib.Geometry.Euclidean.Angle.Unoriented.Basic`, not marked `private`; they're generally meaningful lemmas about angles, not specific to this proof.